frontend/src/concepts/pipelines/content/__tests__/ComputeRunStatusUtils.spec.tsx

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable camelcase*/
```

Comment:
Is it okay to disable camelcase eslint rule for this file?

Reply:
Ya that's fine.
I think we need a tech debt to figure out a way to do better typescript and camel case since this is because of apis we consume.

typescript provides `@typescript-eslint/naming-convention` and i wonder if we can configure it to allow camelcase if a type is already defined.